xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/BlogStyle.xml

```diff
@@ -260,15 +261,6 @@
 .entry-author-label {
   display: inline;
 }
-/* Pseudo-tools visible in the edit mode */
-div.publish-message {
-  background-image: url("$xwiki.getSkinFile('icons/silk/page_white_world.png')");
-  background-position: 6px center;
-}
-div.hide-message {
-  background-image: url("$xwiki.getSkinFile('icons/silk/lock_open.png')");
-  background-position: 6px center;
-}
```

Comment:
AFAIU you've removed this because we're missing a way to get the URL for an icon from the icon set? If so, I think we should this instead.